addon/components/date-range-picker.js

```diff
@@ -59,6 +65,16 @@ export default Ember.Component.extend({
       drops: this.get('drops'),
       opens: this.get('opens')
     });
+
+    this.$('.daterangepicker-input').on('show.daterangepicker', function(ev, picker) {
```

Comment:
Hey @malchak I just came acros a problem with this block of code, I'm getting "Invalid data" when the daterangepicker is shown. What is the exactly reason for this block?